circuit.py

```diff
@@ -330,11 +330,19 @@ def render_latex(self) -> str:
             for operator in operation.ops_list:
                 operator_list += str(operator) 
             
+            # Latex format for phase label (I didnt want to do this in the template file)
             if isinstance(operation, Rotation):
-                phase_list.append((operation.rotation_amount.numerator, operation.rotation_amount.numerator)) 
+                operator_str = '$\\frac{'
+                if abs(operation.rotation_amount.numerator) != 1:
+                    operator_str += str(operation.rotation_amount.numerator)
+                elif operation.rotation_amount.numerator == -1:
+                    operator_str += '-'
```

Comment:
Same comment about minus sign placement here

Reply:
Since there are two of these sections, might be a good idea to refactor into a function too